chrome/script.browserify.js

```diff
@@ -54,6 +54,7 @@ function view() {
         [
           m("input", {
             type: "text",
+            id: "search-field",
```

Comment:
I'm using id for selecting and detecting focus on `input`. It seemed to be better than reading other attributes (like tagName) to me.